apps/pm-user/src/components/molecules/loginForm/LoginForm.js

```diff
@@ -0,0 +1,66 @@
+import React, { useState } from 'react';
+import styles from './loginForm.module.scss';
+import { Button, Input } from '../..';
+
+const LoginForm = ({ onLogin }) => {
+  const [formValues, setFormValues] = useState({
+    emailValue: '',
+    passwordValue: '',
+  });
+
+  function submitForm(e) {
+    e.preventDefault();
+
+    if (formValues) {
+      onLogin(formValues.passwordValue, formValues.emailValue);
+    } else {
+      console.log('email or password can not be empty');
+    }
+  }
```

Comment:
This code does not prevent a user from submitting the login form when there are empty fields.
Please update this code as per Mentor's suggestions.

Reply:
Issue Resolved.